vis/tslint.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
If you delete this file we won't run tests for linting any more. Please keep this, but limit it to just `{ "extends": "../tslint.json" }`.